support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -168,4 +168,36 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		excludeCountriesSubjectToContributionsOnlyAmounts: false,
 	},
+	amazonPay: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			US: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		canRun: () => {
```

Comment:
Good to see this used - we could probably do things like this more. Although maybe speaks to the fact the targetPage is very non-specific?

Reply:
Yeah it's a handy function to have. Yeah `targetPage` is non-specific which is sufficient most of the time, we could probably revisit `pageUrlRegexes` at some point though, as maybe it could be updated so we can target users on specific checkouts, eg. Tier 1, 2 or 3.